.github/workflows/linux.yml

```diff
@@ -33,6 +33,11 @@ jobs:
         if: always()
         run: |
           python3 tools/scripts/rm_whitespace.py
+      - name: Check for Expired Deprecations
+        if: startsWith(github.head_ref, 'release/')
+        run: |
+          egrep -rI "DEPRECATED?:? +`basename ${{ github.head_ref }}`" . || true
+          return $(egrep -rI "DEPRECATED?:? +`basename ${{ github.head_ref }}`" . | wc -l)
```

Comment:
Are you sure `return` works here? (`exit` should work in any case.)

Reply:
Yes, I've tested the behavior on my fork's CI, to make sure this really works!